### DISCUSSION TITLE
which `*>` was inspired by?

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [typelevel/cats#4278] [Q&A] [answered]
Hello, I'm wondering where `*>` comes from. From my investigation, @stew introduced it in [#278](https://github.com/typelevel/cats/pull/278/commits/0bf1ae70169ebc8048cad6faec363c0217343456#diff-02f1e81bbc6a84938b01a4aae3a025b96ef6f6fe9f79186a3ad93934c9600453R19).
Would you tell me that it was inspired by? (i.e. something from Haskell)
Thank you.

**Comment by LukaJCB** [comment, marked as answer]
It's based on the same operator defined in Haskell's `Applicative` typeclass: https://hackage.haskell.org/package/base-4.17.0.0/docs/Control-Applicative.html#v:-42--62-

**Reply by KisaragiEffective** [reply]
Thank you for quick answer :)

**Comment by stew** [comment]
yup, what @LukaJCB said :)
